data/test/tabletserver/exec_cases.txt

```diff
@@ -310,6 +310,41 @@
   "SecondaryPKValues": [2, null]
 }
 
+# on dup, with values() func
+"insert into a (eid, id, name) values (1, 2, 'foo') on duplicate key update name = values(name)"
```

Comment:
We'll need a few more test cases:
* Complex values expression in the update.
* Complex expression in the insert, referenced by a simple values expression in the update.
* Use of bind vars in values.
* Update column mismatch error case.